jc/parsers/net_user.py

```diff
@@ -0,0 +1,441 @@
+r"""jc - JSON Convert `net user` command output parser
+
+
+Usage (cli):
+
+    $ net users | jc --net-user
+    $ net users /domain | jc --net-user
+    $ net users User1 | jc --net-user
+    $ net users User1 /domain | jc --net-user
+
+Usage (module):
+
+    import jc
+    result = jc.parse('net-user', net_user_command_output)
```

Comment:
Should use 'net_user' instead of 'net-user' here. Also, change to 'net-users' if the parser name changes.

Reply:
"net users" works, but its not what is shows in their syntax. I've always used "net user"
`>net /h
The syntax of this command is:

NET
    [ ACCOUNTS | COMPUTER | CONFIG | CONTINUE | FILE | GROUP | HELP |
      HELPMSG | LOCALGROUP | PAUSE | SESSION | SHARE | START |
      STATISTICS | STOP | TIME | USE | USER | VIEW ]`

Reply:
going to leave it as net user unless you have strong opinion for the contrary